docs/resources.md

```diff
@@ -0,0 +1,97 @@
+- aws-cloudwatch 
+
+**EKS: ingress-nginx**:
+Use Nginx as a reverse proxy and load balancer for your cluster. This will create an AWS load balancer (ELB/ALB/NLB) and whenever an ingress is created to route traffic to your application, the controller will make sure the LB is up to date and sending traffic where it needs to go. 
+
+**EKS: External-dns**:
+For any ingresses that are added to route traffic for hosts, external-dns will automatically create DNS records for those hosts and point it to the LB created by the ingress controller. This makes is extremely easy to bring up a new site at a specific domain or subdomain. [How it works][external-dns]
+
+**EKS: Cert-manager**:
+For any ingresses that specify that they need TLS, cert-manager will automatically provision a certificate using Lets Encrypt, and handle renewing it automatically on a regular basis. Alongside external-dns, this allows you to make sure your new domains are always secured using HTTPS. 
+
+To enable cert-manager to create certificate for you using [ClusterIssuer with Zero][zero-cluster-issuer], you would be able to reference it with an annotation in your ingress as follow
+```yml
+## kubernetes ingress annotations
+"cert-manager.io/cluster-issuer": clusterissuer-letsencrypt-production
+## spec > tls > secret_name
+secret_name = "tls-certificate"
+```
+Cert-manager will then create a certificate resource, and use letsencrypt ACME servers to validate and deploy the certificate. Then once all you should should be able to see `kubectl get certificates` as `Ready:true`
+
+**EKS: Kubernetes Dashboard**
+A web-based GUI for viewing and modifying resources in a Kubernetes cluster. The dashboard consists of two parts, a persistence layer and data scraper and the UI itself. 
+- [dashboard-metrics-scraper][k8s-metrics-scraper]
+- [kubernetes-dashboard][k8s-dashboard]
+
+```sh
+# you can access the dashboard with the following command
+kubectl get secret -o json -n kubernetes-dashboard $(kubectl get secret -n kubernetes-dashboard | grep dashboard-user-token | awk '{print $1}') | jq -r .data.token | base64 -D | pbcopy && \
+open "http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/#/login" && kubectl proxy
+```
+
+**EKS: Metrics-server**
+[A collector][k8s-sigs-metrics] of cluster-wide resource metrics. Used by things like HorizontalPodAutoscaler to determine the current usage of pods. Also allows the `kubectl top` command
+
+**EKS: Cluster-autoscaler**
+Kubernetes deployment monitoring metrics gathered by metrics-server, interacts with AWS autoscaling group to scale the cluster. Scaling the cluster based on resources expenditure and [defined rules][asg-config].
+
+**RDS**:
+AWS Relational database services, we provision a postgres DB for your application that is within your VPC within your EKS cluster's worker-security group(accessible by your worker-nodes), the database is also pre-configured to be monitored with cloudwatch.
```

Comment:
"we provision a postgres DB for your application that is within your VPC and accessible by your EKS worker nodes. The database is also pre-configured to be monitored with cloudwatch."